.gitignore

```diff
@@ -1,7 +1,7 @@
 __pycache__/
 .pytest_cache/
 .cache/
-
+.vscode/
```

Comment:
Do we really want to do this? It won't remove existing settings but it will ignore any further changes right?

I'm assuming you added this because of [Kamil's comment](https://github.com/v7labs/darwin-py/pull/225#discussion_r725867447). If that's the case, then I don't think this is the correct solution.

Instead of adding `"python.pythonPath": "/home/zuppif/miniconda3/bin/python"` to the project's VSCode settings, you can add it to your own VSCode's settings for Python.

Alternatively, if you want to set the Python path for this project only, then you can add this line to a [global `.gitignore` file in your system](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer), or [to a file under `.git/info/exclude`](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file). Both these situations will ignore changes to these files in your system, but still allow others to keep changing the file in the repo.